src/scripts/active-inactive-orgunit/index.js

```diff
@@ -98,69 +94,70 @@ OrganisationUnitActivator.prototype.processDataElementGroup = function (dataElem
         );
 
         //Process every dataElements
-        dataElements.forEach(function (dataElement) {
-            dataElement.dataElementGroup = dataElementGroup;
+        dataElements.forEach(function (dataElement) { 
             console.info("\nConfig:\n", JSON.stringify(dataElement, null, "\t"));
-            _this.prepareDataElement(dataElement);
-            _this.processDataElements(dataElement);
+            var isDataElementValid = _this.prepareDataElement(dataElement);
+            if (isDataElementValid) {
+                _this.processDataElements(dataElement, dataValues);
+            }
         });
         _this.asyncCalls--;
     });
 };
 
 /**
  * Builds the dataElement attributes
+@param dataElement The active dataElement
  */
 OrganisationUnitActivator.prototype.prepareDataElement = function (dataElement) {
-
-    var _this = this;
+    var attributeUids = this.attributeUids;
     console.log("\nPreparing dataElement:" + dataElement.id);
     dataElement.attributeValues.forEach(function (attributeValue) {
-        if (attributeValue.attribute.id == _this.attributeUids.LEVEL) {
+        if (attributeValue.attribute.id == attributeUids.LEVEL) {
             dataElement.level = attributeValue.value;
         }
-        if (attributeValue.attribute.id == _this.attributeUids.PARENT) {
+        if (attributeValue.attribute.id == attributeUids.PARENT) {
             dataElement.parent = attributeValue.value;
         }
-        if (attributeValue.attribute.id == _this.attributeUids.ORGUNITGROUP) {
+        if (attributeValue.attribute.id == attributeUids.ORGUNITGROUP) {
             dataElement.orgUnitGroup = attributeValue.value;
         }
-        if (attributeValue.attribute.id == _this.attributeUids.PERIOD) {
+        if (attributeValue.attribute.id == attributeUids.PERIOD) {
             dataElement.periods = attributeValue.value;
         }
     });
-};
 
+    if ((dataElement.parent == undefined || dataElement.level == undefined) && dataElement.orgUnitGroup == undefined) {
+        console.error("Invalid dataElement organisation unit attributes", " DataElement:" + dataElement.id);
```

Comment:
Remove the comma and join the two strings. It is a bit strange how you are printing this error message